.gitignore

```diff
@@ -0,0 +1,12 @@
+# Node build artifacts
+node_modules/
+npm-debug.log
+
+# Local development
+*.env
+*.dev
+.DS_Store
+
+# Docker
+Dockerfile
+docker-compose.yml
```

Comment:
It's best to add a new line at the end of each file

Reply:
got it